web/frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "@improbable-eng/grpc-web": ">=0.15",
     "@viamrobotics/prime": ">=0.5",
     "@viamrobotics/rpc": ">=0.2",
-    "@viamrobotics/sdk": "0.21.0",
+    "@viamrobotics/sdk": "0.22.0-b.0",
```

Comment:
Why is the name different now? What happened to the release candidate? What is `-b.0`?

Reply:
i believe this is beta version which is going to be released tomorrow  @stuqdog  @njooma

Reply:
i selected latest fyi @njooma  in the bump rc workflow

Reply:
This is fine -- this was just released and there are no changes between when this was released and today